### PR TITLE
mod: support for noir

### PR DESCRIPTION
**Pull request by Adebara123**

### CODE REVIEW

File: src/utils/helpers.rs [hunk 1]
```diff
@@ -103,6 +103,14 @@ pub fn is_choco_installed() -> bool {
     check_output(output)
 }
 
+pub fn is_nargo_installed() -> bool {
```
Comment (casweeney) [inline src/utils/helpers.rs:106]: let the name of the function be `is_noir_installed`

File: src/utils/helpers.rs [hunk 2]
```diff
@@ -421,3 +429,20 @@ pub fn install_forge() -> Result<(), Box<dyn Error>> {
         Ok(())
     }
 }
+
+
+pub fn install_nargo() -> Result<(), Box<dyn Error>> {
```
Comment (casweeney) [inline src/utils/helpers.rs:434]: let the function name be `install_noir` nargo is what comes with noir installation just like cargo comes with rust installation.

File: src/lib.rs [hunk 3]
```diff
@@ -184,7 +184,15 @@ pub fn resolve(args: ClapperArgs) -> Result<(), Box<dyn Error>> {
                             return  Err(e);
                         }
                     }
-                }
+                },
+                InstallSubCommand::Noirup => {
```
Comment (casweeney) [inline src/lib.rs:188]: Let's just use `Noir` instead of `Noirup`

File: src/utils/command_arguments.rs [hunk 4]
```diff
@@ -95,7 +95,9 @@ pub enum InstallSubCommand {
     /// Installs Forge
     Forge,
     /// Installs Starkli
-    Starkli
+    Starkli,
+    /// Installs Nargo
+    Noirup
```
Comment (casweeney) [inline src/utils/command_arguments.rs:100]: Let's just keep this as `Noir` instead of `Noirup` for simplicity.

**Review by casweeney — APPROVED** [review]

File: src/utils/helpers.rs (hunk 1 above)
Comment (Adebara123) [reply src/utils/helpers.rs:106]: nargo is the name of the binary, and that is what is used in the official documentation.

**Review by casweeney — APPROVED** [review]

**Pull request merged**